src/v2/api/controllers/AuthController.ts

```diff
@@ -143,6 +144,25 @@ export class AuthController {
     return this.authService.refresh(req.user);
   }
 
+  @ApiBearerAuth()
```

Comment:
add a description of the 401 exception to Swagger

Reply:
I cannot find any place where this exception is thrown